platform/windows/detect.py

```diff
@@ -198,7 +198,7 @@ def configure_msvc(env, manual_msvc_config):
         env.Append(LINKFLAGS=["/OPT:REF"])
 
     elif env["target"] == "debug":
-        env.AppendUnique(CCFLAGS=["/Z7", "/Od", "/EHsc"])
+        env.AppendUnique(CCFLAGS=["/Z7", "/Of", "/EHsc"])
```

Comment:
That's clearly wrong. `/Od (Disable (Debug))`

Reply:
Ouch, nice catch. Silly codespell.